src/Microsoft.Health.Dicom.Api/Features/Exceptions/ExceptionHandlingMiddleware.cs

```diff
@@ -122,6 +123,7 @@ private IActionResult GetContentResult(HttpStatusCode statusCode, string message
 
         protected internal virtual async Task ExecuteResultAsync(HttpContext context, IActionResult result)
         {
+            EnsureArg.IsNotNull(result, nameof(result));
```

Comment:
Ensure context? I realize it's probably not explicitly required, but we seem to be ensuring the `HttpContext` everywhere else.

Reply:
addressed